README.md

```diff
@@ -152,3 +152,7 @@ module "spectral_lambda_integration" {
 8. `lambda_iam_role_arn` - Amazon Resource Name (ARN) specifying the role.
 9. `lambda_iam_role_name` - Name of the role.
 10. `secrets_arns` - Arns of created secrets in secrets manager.
+
+## Support
```

Comment:
Adding a changelog file that lists versions, describes the changes, and breaks changes per version is essential, and can avoid a lot of questions.